src/main/java/de/tum/in/www1/artemis/repository/iris/IrisChatSessionRepository.java

```diff
@@ -26,9 +26,9 @@ public interface IrisChatSessionRepository extends JpaRepository<IrisChatSession
      */
     @Query("""
                 SELECT s
-                FROM IrisChatSession s
+                    FROM IrisChatSession s
```

Comment:
Why do you indent the `FROM` here? You do not do this for other queries like `countLlmResponsesOfUserWithinTimeframe` (or in the Hestia PR)

Reply:
Because humans are bad at repetitive tasks ^^ Thanks for noticing